homeassistant/components/neato/services.yaml

```diff
@@ -0,0 +1,18 @@
+neato_custom_cleaning:
```

Comment:
This service call actually exists in the `vacuum` domain it does not exist in the `neato` domain.  You will need to modify this service call so its under the `neato` domain in order for this PR to work.